ja.po: ch_ops_backup_recovery.xml

amotoki opened the pull request.

amotoki committed:
ja.po: ch_ops_backup_recovery.xml

irixjp commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,334 @@
+"While OpenStack is composed of many components and moving parts, backing up "
+"the critical data is quite simple."
+msgstr ""
+"OpenStackは多くのコンポーネントからなり、それらは交換可能と複雑ですが、クリ"
```
While OpenStack is composed of many components and moving parts
「OpenStackには多くのコンポーネントと可動部が含まれているが」のほうがしっくりくる感じですねかね。

amotoki replied:
「可動部」としてもまだ何のことかよく分からないです。元の moving parts の意味自体がよく分かっていません。
コンピュータシステムで moving parts という表現は個人的には初めて見ました。
「動的に変化する部分、要素」といった感じでしょうか。 ご意見求む！

yosshy replied:
moving parts は機械的な可動部分の事で、パソコンで言えば HDD や冷却ファン、キーボード、LCDのヒンジ等です。
バックアップの章の文脈では、OpenStack ソフトウェア以外の関連ソフトウェア（DB、AMQP、memcached、ハイパーバイザ、他）を指しているのではないでしょうか。
ぶっちゃけ、原文を書いた人間に聞くのが一番です。

amotoki replied:
訳し方については議論がまだあると思います。必要なら https://github.com/openstack-ja/openstack-manuals/issues/50 で議論を続けましょう。

とりあえず以下のようにしてみました。
「OpenStack は多くのコンポーネントから構成され、注意を払うべき箇所もたくさんありますが、大事なデータのバックアップは非常に単純です。」

irixjp commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,334 @@
+"throughout your storage cluster."
+msgstr ""
+"<code>/etc/swift</code>は非常に重要ですのでバックアップが必要です。このディレ"
+"クトリは、RingファイルやRingバッファファイルなどの消失した場合はクラスター上"
```
「Ringファイルや、Ringビルダーファイル」

ですかね。

Commit:
s/ブロック・ストレージ/ブロックストレージ/

amotoki committed:
Update translations

Update translation of backup of ring and ring builder files.

s/compute ノード/コンピュートノード/
s/cloud コントローラー/クラウドコントローラー/

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,335 @@
+"Availability as well as backups."
+msgstr ""
+"もし、いかなるデータロスも許されない場合、バックアップに加えてHAも検討すべき"
+"です。"
```
他では High Availability と書かれていることが多いので、それに合わせた方がよいと思います。
「高可用性」でしょうか。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,335 @@
+msgstr ""
+"OpenStackバックアップポリシーを作成する際、標準的なバックアップのベストプラク"
+"ティスが適用できます。例えば、どの程度の頻度でバックアップを行なうかは、デー"
+"タ損失からどの程度の期間でリカバリーする事と密接に関連しています。"
```
「どのくらい早くデータロスから復旧させる必要があるかに密接に関連しています。」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,335 @@
+"recovery testing)."
+msgstr ""
+"バックアップポリシーと同様に大事なことは、リカバリーポリシー(もしくはリカバ"
+"リーのテスト)を定めることです"
```
at least の訳が抜けています。
「リカバリーポリシーです (少なくともリカバリーのテストは必要です) 。」などでしょうか。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,335 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_ops_backup_recovery.xml:53
+msgid "Database Backups"
+msgstr "データベース バックアップ"
```
データベースのバックアップ

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,335 @@
+"ここでは、Cloud コントローラーに MySQL サーバーを利用しています。このMySQL "
+"サーバーは Nova, Glance, Cinder, そして Keystone のデータベースを保持していま"
+"す。全てのデータベースが一つにある場合、データベースバックアップは非常に容易"
+"となります。"
```
「参考アーキテクチャーでは、クラウドコントローラーを MySQL サーバにしています。」

「一つにある場合」→「一ヶ所にある場合」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,335 @@
+"の例外がコンピュートノードにある <code>/var/lib/nova/instances</code> サブディレ"
+"クトリです。このサブディレクトリは実行中のインスタンスの KVM イメージを含みま"
+"す。すべてのインスタンスのバックアップコピーを維持したければ、このディレクト"
+"リをバックアップしたいかもしれません。多くの場合において、これを実行する必要"
```
「このディレクトリには実行中のインスタンスのKVMイメージが置かれています。
このディレクトリをバックアップしたいと思うのは、すべてのインスタンスのバックアップコピーを保持する必要がある場合だけでしょう。」
you would only want to back up this directory if .... なので、きちんと訳した方がよいと思います。
また、directory contains XXX は XXX が置かれています、とした方が自然になります

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,335 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_ops_backup_recovery.xml:108
+msgid "Image Catalog and Delivery"
+msgstr "イメージカタログと配布"
```
「イメージカタログと配布」が Glance というのは分かりにくいですね・・・タイトルだけ見ると。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,335 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_ops_backup_recovery.xml:179
+msgid "Now you can import a previously backed up database:"
+msgstr "バックアップしたファイルをインポートします。"
```
「以前にバックアップしたデータベースをインポートします」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,335 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_ops_backup_recovery.xml:182
+msgid "As well as restore backed up nova directories:"
+msgstr "同様にnovaディレクトリをリストアします。"
```
バックアップした nova のディレクトリをリストアします。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_backup_recovery.xml.ja.po:
```diff
@@ -0,0 +1,335 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_ops_backup_recovery.xml:177
+msgid "Once that's done, stop MySQL:"
+msgstr "実行後、MySQLを停止します。"
```
実行後→その後

amotoki committed:
Remove unnecessary newline(s)

amotoki commented:
間違えて Close を押してしまったみたいです。

amotoki committed:
Update translations in ch_ops_backup_recovery

amotoki committed:
Update trans of "moving parts" in ch_ops_backup_recovery.xml
